README.md

```diff
@@ -169,7 +173,7 @@ In some cases, you may want to limit the amount of logs the Datadog buildpack is
 
 To limit the log output of the buildpack, set the `DD_LOG_LEVEL` environment variable to one of the following: `TRACE`, `DEBUG`, `INFO`, `WARN`, `ERROR`, `CRITICAL`, `OFF`.
 
-```
+```text
```

Comment:
shouldn't this be shell?

Reply:
I put `text` by default in order to fix markdown linting issue but yeah we could do shell, i'm going to push and update :)